inference-engine/tests/functional/plugin/shared/include/single_layer_tests/interpolate.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+#include <memory>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+
+namespace LayerTestsDefinitions {
+
+typedef std::tuple<
+        std::set<size_t>,                            // Axes
+        ngraph::op::Interpolate::InterpolateMode,    // InterpolateMode
```

Comment:
please check v3 operation - it contains some additional parameters which are not used in current implementation

Reply:
Fixed